src/plugins/saved_objects_management/server/routes/find.ts

```diff
@@ -45,17 +45,24 @@ export const registerFindRoute = (
       },
     },
     router.handleLegacyErrors(async (context, req, res) => {
+      const { query } = req;
       const managementService = await managementServicePromise;
-      const { client } = context.core.savedObjects;
-      const searchTypes = Array.isArray(req.query.type) ? req.query.type : [req.query.type];
-      const includedFields = Array.isArray(req.query.fields)
-        ? req.query.fields
-        : [req.query.fields];
+      const { getClient } = context.core.savedObjects;
+
+      const searchTypes = Array.isArray(query.type) ? query.type : [query.type];
+      const includedFields = Array.isArray(query.fields) ? query.fields : [query.fields];
+
       const importAndExportableTypes = searchTypes.filter((type) =>
         managementService.isImportAndExportable(type)
       );
 
+      const includedHiddenTypes = importAndExportableTypes.filter((type) =>
+        managementService.isHidden(type)
+      );
+
+      const client = getClient({ includedHiddenTypes });
```

Comment:
This is the one I'm unsure about.

@rudolf @kobelb, from https://github.com/elastic/kibana/issues/82027, do we want those hidden types to be exportable from the UI, or only using the http API?

(Note that if we want to adapt the UI, this endpoint is not enough, e.g `src/plugins/saved_objects_management/server/routes/scroll_count.ts`)

Reply:
Yeah, as long as `management.importableAndExportable === true`

With https://github.com/elastic/kibana/issues/82027#issuecomment-738467282 I believe Brandon meant these will show up in the UI because they're being returned by the `api/kibana/management/saved_objects/_find` endpoint.

Reply:
Thank you both ❤️ I've updated the rest of the SOM apis to show the hidden yet importable/exportable types. This direction makes sense to me